plugins/webhooks/index.js

```diff
@@ -254,52 +335,57 @@ function pullRequestEvent(pluginOptions, request, reply, parsed) {
     const pipelineFactory = request.server.app.pipelineFactory;
     const userFactory = request.server.app.userFactory;
     const { hookId, action, checkoutUrl, branch, sha, prNum, prRef,
-        prSource, username, scmContext, changedFiles } = parsed;
+        prSource, username, scmContext, changedFiles, type } = parsed;
     const fullCheckoutUrl = `${checkoutUrl}#${branch}`;
-    let scmToken = null;
+    const scmConfig = {
```

Comment:
This might need `prNum` to get pull request via octokit.
If you add this `prNUm` here, you will get error when you call scm.getBranchList function with scmConfig which include `prNum` info. Therefore please write code which does not send `prNum` to scm.getBranchList.

Reply:
Thank you!
I fixed to add `prNum` to scmConfig in `createPREvents`.